ocfweb/announcements/templates/announcements/2017-03-20-hiring.html

```diff
@@ -0,0 +1,121 @@
+            </p>
+
+            <h3>Operations Assistants</h3>
+            <p>
+                Operations Assistants will have the following responsibilities:
+
+                <ul>
+                    <li>Staffing the front desk</li>
+                    <ul>
+                        <li>Maintaining lab security</li>
+                        <li>Providing help to OCF members using the lab</li>
+                        <li>Carrying out regular lab maintenance</li>
+                    </ul>
+                    <li>Working on projects that benefit the OCF</li>
+                    <ul>
+                        <li>Marketing and outreach</li>
+                        <li>Collecting feedback on OCF services</li>
+                        <li>Organizing events taking place in the lab</li>
+                        <li>Researching and applying for additional sources of
+                            funding for the OCF</li>
+                    </ul>
+                    <li>Attending regular meetings with the Operations Manager
+                        and the other Operations Assistants</li>
+                </ul>
+
+                This job position will be paid $13.75 per hour, but will also be
+                less work and more flexible than the Operations Manager
+                position. See the
+                <a href="https://docs.google.com/document/d/1oS3ma415LbtuyeEuuoucWKYLcWOJaWmzhv2nIs5f718/edit">full job description</a>
+                for more details.
+            </p>
+
+            <h3>Operations Manager</h3>
+            <p>
+                The Operations Manager will have the same responsibilities as
+                the Operations Assistants (see above), except they will spend
+                less time staffing the front desk of the OCF, and instead will
+                spend the remaining time performing more managerial duties,
+                such as leading meetings with the Operations Assistants,
+                delegating projects, and scheduling shifts for the OCF front
+                desk. This job will involve more work than the Operations
+                Assistant role, but it is also paid more, at $15.75 per hour.
+                See the
+                <a href="https://docs.google.com/document/d/1oS3ma415LbtuyeEuuoucWKYLcWOJaWmzhv2nIs5f718/edit#bookmark=id.im3qxqp4zku2">full job description</a>
+                for more details.
+            </p>
+
+            <h3>Application</h3>
+            <p>
+                <a href="https://docs.google.com/a/berkeley.edu/forms/d/e/1FAIpQLSf4sSEdCg6gDbgYJPPvxtXKIcuveygjCBFg_pWuwWoRIFXpXQ/viewform">Apply online</a>
+                for either position. Applications close on Tuesday, April 7th
+                at 11:59 PM, so make sure to apply by then if you would like to
+                be considered.
+            </p>
+            <p>
+                We will let you know if you have been selected for a 30 minute
```

Comment:
Ah, good point, I forgot they were only 20 minutes, probably best to put that, don't want to be too intimidating or anything. Not that 30 minutes should be especially intimidating or anything.